src/test/java/ClientIDServletTest.java

```diff
@@ -22,11 +21,13 @@
 /** Test Client ID Servlet to ensure response contains a defined OAuth 2.0 client ID */
 @RunWith(JUnit4.class)
 public final class ClientIDServletTest extends ServletTestBase {
-  private static final ClientIDServlet servlet = new ClientIDServlet();
+
+  private static final String CLIENT_ID = "sampleValue";
+  private static final ClientIDServlet servlet = new ClientIDServlet(CLIENT_ID);
 
   @Test
   public void responseContainsClientId() throws Exception {
     servlet.doGet(request, response);
-    Assert.assertTrue(stringWriter.toString().contains(AuthenticationVerifier.CLIENT_ID));
+    Assert.assertTrue(stringWriter.toString().contains(CLIENT_ID));
```

Comment:
This seems to be one of my previous commits. Try rebasing and merging with master.

Reply:
`Resolved`